### PR TITLE
Add QA Team doc

### PR DESCRIPTION
**Pull request by suspect15**
Adds a doc outlining the QA team's equities to give users a clear understanding of its responsibilities and authorities.  It explains QA's processes for making changes to active and inactive developer sets.  All content is sourced from the internal QA responsibilities and authorities matrix.

### CODE REVIEW

File: docs/general/teams/qa-team.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Current team members can be found on the [About Us](/general/about-us.html#quality-assurance-team) page.
+
+## QA's Responsibilties and Authorities
+
+QA's responsibilities include maintaining inactive developer sets, ensuring all sets conform to the writing policy, add necessary achievement protections, repairing malfunctioning leaderboards, managing unstable sets, overseeing several sub-teams, onboarding people into the [Junior Developer Program](/developer-docs/jr-dev-rules.html), managing hashes, and maintaining content guidelines.
+
+QA has the authority to fix any issues in sets with no active developers.  QA may also fix issues in sets with active developers under the following processes:
+
+No advance coordination required:
+- Set wide exploitation. QA must immediately contact developer to hand off repair and repromotion responsibility
```
Comment (wescopeland) [inline docs/general/teams/qa-team.md]:
```suggestion
- Set-wide exploitation. QA must immediately contact the developer to hand off repair and repromotion responsibility
```

File: docs/general/teams/qa-team.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+- Remove custom player input from rich presence
+
+Developer must be contacted, QA may remediate if no response is received within 7 days for:
+- Demote set due to instability (generally 10 or more tickets)
+- Fix achievement title or description grammar, spelling, capitalization or wording error
```
Comment (wescopeland) [inline docs/general/teams/qa-team.md]:
```suggestion
- Fix achievement title or description grammar, spelling, capitalization, or wording error
```

File: docs/general/teams/qa-team.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+- Demote achievement due to lack of protection or exploit
+- Adjust set progression types
+- Fix leaderboard or rich presence grammar, spelling, capitalization or wording error
```
Comment (wescopeland) [inline docs/general/teams/qa-team.md]:
```suggestion
- Fix leaderboard or rich presence grammar, spelling, capitalization, or wording error
```

File: docs/general/teams/qa-team.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+## QA's Responsibilties and Authorities
+
+QA's responsibilities include maintaining inactive developer sets, ensuring all sets conform to the writing policy, add necessary achievement protections, repairing malfunctioning leaderboards, managing unstable sets, overseeing several sub-teams, onboarding people into the [Junior Developer Program](/developer-docs/jr-dev-rules.html), managing hashes, and maintaining content guidelines.
+
+QA has the authority to fix any issues in sets with no active developers.  QA may also fix issues in sets with active developers under the following processes:
+
+No advance coordination required:
+- Set wide exploitation. QA must immediately contact developer to hand off repair and repromotion responsibility
+- Remove custom player input from rich presence
```
Comment (wescopeland) [inline docs/general/teams/qa-team.md]: Recommend linking the policy here, eg:

```md
- Remove custom player input from rich presence to follow [the current policy](/some link)
```

File: docs/general/teams/qa-team.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
+---
+title: QA Responsibilities
```
Comment (wescopeland) [inline docs/general/teams/qa-team.md]: Title should probably match the name of the link: "Quality Assurance". Recommend also renaming the file to _quality-assurance.md_.

File: docs/general/teams/qa-team.md (hunk 4 above)
Comment (suspect15) [reply docs/general/teams/qa-team.md]: Added as suggested

File: docs/general/teams/qa-team.md (hunk 5 above)
Comment (suspect15) [reply docs/general/teams/qa-team.md]: Agreed, this was a relic of a previous iteration.  Updated.

**Review by wescopeland — APPROVED** [review]

**Pull request merged**